resteasy-spring-boot-starter-test/src/test/java/org/jboss/resteasy/springboot/LogbackTestApplicationListener.java

```diff
@@ -37,7 +39,10 @@ protected void append(ILoggingEvent event) {
                 return;
             }
             Level level = event.getLevel();
-            if ((level.equals(Level.WARN) || level.equals(Level.ERROR)) && !event.getMessage().startsWith(SCANNING_WARNING)) {
+            if ((level.equals(Level.WARN) || level.equals(Level.ERROR))
+                    && !event.getMessage().startsWith(SCANNING_WARNING)
+            && !event.getMessage().startsWith("InetAddress.getLocalHost")
+            && !event.getLoggerName().equals("org.apache.tomcat.util.modeler.Registry")) {
```

Comment:
This is because of the springboot tomcat dependency upgrade and now it's embedded tomcat server will printout some relative warning messages.

Reply:
Maybe a comment in the code could help in the future

Reply:
okay!